back-nest/src/sсrapers/scrapers.controller.ts

```diff
@@ -1,19 +1,14 @@
-import { Controller, Param, Get, HttpCode, Logger } from '@nestjs/common';
+import { Controller, Param, Post, HttpCode } from '@nestjs/common';
 
-import { ScraperServiceFactory, ServiceType } from './ScraperServiceFactory';
+import { ScraperGateway } from '../ws/ws.gateway';
 
 @Controller('scraper')
 export class ScraperController {
-  private readonly logger = new Logger(ScraperController.name);
-
-  constructor(private readonly scraperServiceFactory: ScraperServiceFactory) {}
+  constructor(private readonly scraperGateway: ScraperGateway) {}
 
   @HttpCode(204)
-  @Get(':serviceType')
+  @Post(':serviceType')
```

Comment:
Use `kebab-case` naming style in your endpoints. Apply this to all following requests in projects